src/index.js

```diff
@@ -12,6 +12,7 @@ import type {Config} from './Config'
 import initializeServices from './services/initializeServices'
 import commander from 'commander'
 import cosmiconfig from 'cosmiconfig'
+import startCronJobs from './cronjobs'
```

Comment:
The file should be named `startCronJobs` because it exports a function as default. Does a service make sense here?

Reply:
I don't think a service is necessary for this.